CHANGELOG.md

```diff
@@ -1,16 +1,37 @@
-## 2.2.0 - 2024-06-25
-[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
-## 2.1.7 - 2023-09-14
-- Removing mm9, updating mm10, and making ploidy file optional
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [2.2.0] - 2024-06-25
+### Added
+- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - Add vidarr labels to outputs (changes to medata only).
+
+## [2.1.7] - 2023-09-14
+### Changed
+- Removing mm9, updating mm10, and making ploidy file optional.
+
 ## [2.1.6] - 2023-05-30
-- Optmizing issues specified in GRD-544
-## Unreleased - 2021-11-10
-[GP-2887](https://jira.oicr.on.ca/browse/GP-2887) Making RT tests more robust
-## 2.3 - 2021-01-08
- - Added genome size and reference parameters, ploidy file is optional. Modules used by default are R 4.0 - compliant, to enable support for newer assemblies (hg38, mm11)
-## 2.2 - 2020-10-15
- - Added an ability to produce .png for the contour and genome plots showing the effect of gamma value, along with a summary plot and a rmarkdown .pdf with all new plots together
-## 2.1 - 2020-06-26
- - Added window, gamma, female and cancerType parameters for sequenza. Fixed an issue in R script, made more robust when dealing with challenging data
-## 2.0 - 2020-05-07
- - Initial Release
+### Fixed
+- Optmizing issues specified in GRD-544.
+
+## [Unreleased] - 2021-11-10
```

Comment:
This CHANGELOG needs additional work as records for many tags are missing. The dates are known (see Code->Tags) but the changes need to be checked in pull request history/Jira tickets